cmd/web.go

```diff
@@ -495,6 +495,7 @@ func runWeb(ctx *cli.Context) {
 					Post(bindIgnErr(auth.NewWikiForm{}), repo.NewWikiPost)
 				m.Combo("/:page/_edit").Get(repo.EditWiki).
 					Post(bindIgnErr(auth.NewWikiForm{}), repo.EditWikiPost)
+				m.Post("/:page", bindIgnErr(auth.NewWikiForm{}), repo.DeleteWikiPagePost)
```

Comment:
It's a POST, so we shouldn't need to add /delete to it if we follow restful actions, unless you want it that way for consistency with the other names ie `_edit`